CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
-# 1.2.9 (2021-04-04)
+# 1.2.10 (2022-05-20)
+## Edge Hub
+### Bug Fixes
+* Configurable task for cancelling hanging upstream calls( [12b52ba](https://github.com/Azure/iotedge/commit/12b52babe6fba24b136971a5d1ce1b8df387b1d7) )
+
+## aziot-edge
+### Bug Fixes
+* Adding error message when listener crashes( [f2e5a47](https://github.com/Azure/iotedge/commit/f2e5a47bb3fd109b5523cb2ae9849401f0fcc99d) )
+* Fix exit code when restarting due to reprovision( [d7d98d0](https://github.com/Azure/iotedge/commit/d7d98d0b606018c6cf5a60f15c44c01c663e0655) )
+* Mariner 2.0 Package Build for IoTEdge( [63273b1](https://github.com/Azure/iotedge/commit/63273b1a6b543eee64d1cb6a8a8fdd8caed47c63) )
+ 
+### Features
+*  Flatten additional properties of metrics ( [4983128](https://github.com/Azure/iotedge/commit/12b52babe6fba24b136971a5d1ce1b8df387b1d7) )
```

Comment:
This one seems like a bug fix for EdgeAgent.

Reply:
Checked with Varun. its a feature for EA